code/modules/food/food.dm

```diff
@@ -20,6 +20,9 @@
 	handle_name_change(usr)
 
 /obj/item/reagent_containers/food/proc/handle_name_change(var/mob/living/user)
+	if(!isliving(user) || user.stat == DEAD)
```

Comment:
I would say this should be limited to humans and silicons, and so prefer
`user.stat == DEAD || !(ishuman(user) || isrobot(user))`
just to eke out that tiny stat-first and short circuit benefit